.ci/renewable-deploy/renewal-script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -ex
+
+cd /code
+
+gcloud config set project ${GOOGLE_CLOUD_PROJECT_ID}
+gcloud config set compute/zone ${GOOGLE_CLOUD_ZONE}
+gcloud auth activate-service-account --key-file /tmp/infrastructure/google-cloud/service-account.json
+gcloud container clusters get-credentials ${GOOGLE_CLOUD_CLUSTER_NAME}
+
+# Find running webserver and postgres
+## TODO - remove listing pods
```

Comment:
What with this todo?

Reply:
removed in https://github.com/shopsys/demoshop/pull/84/commits/f3b4daf6eaf014cb7c1f62dc707eb88b914b1622